### DISCUSSION TITLE
Can not find the fcitx5-configtool on Ubuntu

### DISCUSSION THREAD

**Discussion by thanhpp** [fcitx/fcitx5#603] [Q&A] [answered]
Can not install fcitx5-configtool on Ubuntu

**Comment by wengxt** [comment, marked as answer]
which version of ubuntu? if prior 22.04, I don't suggest using the fcitx5 from repo, it's just too old.

These are the packages you're looking for.
https://packages.ubuntu.com/jammy/fcitx5-config-qt  
https://packages.ubuntu.com/jammy/kde-config-fcitx5
